lib/swc-to-babel.js

```diff
@@ -112,7 +112,9 @@ module.exports = (node, source) => {
             if (path.type === 'ParenthesisExpression')
                 return convertParenthesisExpression(path);
             
-            if (/^(KeyValue|KeyValuePattern|AssignmentPattern)Property$/.test(path.type))
+            if (
+                /^(KeyValue|KeyValuePattern|AssignmentPattern)Property$/.test(path.type)
+            )
```

Comment:
Could you please write it in one line and add test for it?

Reply:
> Could you please write it in one line and add test for it?

I am not really sure how I would test it. And also my formatter really seems to screw up the code

Reply:
ESLint, just add one more test with one argument.